android/quest/src/main/res/values/strings.xml

```diff
@@ -7,11 +7,13 @@
     <string name="run_cql">Run CQL</string>
     <string name="run_cql_log">CQL FULL LOG</string>
     <string name="run_cql_output">CQL OUTPUT DATA</string>
-  
+    <string name="run_control_test">Run Control Test</string>
```

Comment:
This certainly and line 16 need to move to a loaded config file

Reply:
done